src/sentry/uptime/consumers/results_consumer.py

```diff
@@ -375,16 +373,11 @@ def handle_active_result(
                 "uptime_active_resolved",
                 extra={
                     "project_id": project_subscription.project_id,
-                    "url": project_subscription.uptime_subscription.url,
+                    "url": uptime_subscription.url,
                     **result,
                 },
             )
-        # TODO(epurkhiser): Dual until we're only reading the uptime_status
-        # from the uptime_subscription.
-        project_subscription.update(
-            uptime_status=UptimeStatus.OK, uptime_status_update_date=django_timezone.now()
-        )
```

Comment:
Can we keep this around for a while? Just to make sure we can roll back if needed